### DISCUSSION TITLE
How to setup multiple Carousel components in the same page?

### DISCUSSION THREAD

**Discussion by harryps** [mantinedev/mantine#4887] [Q&A]
Hi, I'm trying to setup multiple carousel components on the same page, where each carousel is inside a tab. The problem I'm having is that  only the first one, in the first tab is working. I found out that this is possible with the embla-carousel https://github.com/davidjerleke/embla-carousel/issues/28. What would be the best approach to do this with the Mantine UI component?

**Comment by rtivital** [comment]
There is no limitation on how many carousel components you can render on the same page, for example the documentation page of Carousel component has 10+ different carousels – https://mantine.dev/others/carousel/

**Reply by yangyangshasha** [reply]
Same I
if Carousel in tab, Carousel can't work in second Tabs.Panel.

**Reply by yangyangshasha** [reply]
just work in defaultValue Tab
